content/Watsonx/WatsonxGov/Level-4/201.md

```diff
@@ -0,0 +1,178 @@
+
+   Reservation status is available at https://techzone.ibm.com/my/reservations.
+
+   You must now wait until the environment has been provisioned and the Cloud Pak Deployer service installs the required software before moving on to the next section.
+
+### Accessing the watsonx.governance Environment
+
+1. Navigate to your [TechZone reservations](https://techzone.ibm.com/my/reservations) page.
+
+2. Locate and click on the tile for the environment you reserved in a previous step.
+
+  ![](../images/201/techzone-res-tile.png)
+
+3. Copy and paste the reservation **Username** and **Password** values from the reservation screen into a text file for later use. Then click on the **Open your IBM Cloud environment** button. A new tab opens with the Red Hat OpenShift log in form.
+
+  ![](../images/201/techzone-res-env-select.png)
+
+4. If given the option to log in with **kube:admin** credentials or **IBMid** credentials, click on **kube:admin**.
+
+5. Using the values you copied in step 3, log in to the OpenShift cluster.
+
+  Now that you have successfully logged into your cluster, you will need the credentials to log into Cloud Pak for Data.
+
+6. Click the **Pipelines** menu on the left side of the screen to expand it and then click the **Pipelines** item from the list.
+
+  ![](../images/201/openshift-pipelines.png)
+
+7.  Click the **PipelineRuns** tab to select it. From the list of pipeline runs, click the link for **cloud-pak-deployer-xxx**. It will likely be the only item in the list. 
+
+  > **Note:** if the run is not listed as **Completed** then you will need to wait for it to finish. This pipeline typically takes **4 hours or longer** to run and install all the required software.
+
+  ![](../images/201/openshift-pipelineruns.png)
+
+8. Click on the **Logs** tab. Click on the last item in the list on the left, titled **update-configmap-success-iam**. **Note** that if this item does not appear in the list, then either the software installation has failed (if the pipeline run is complete) or has not fully finished. Copy and paste the **Console Route**, **Username**, and **Password** information from the log window into a text file. Throughout the remainder of these labs, you will use these credentials whenever you are instructed to log into Cloud Pak for Data or watsonx.
+
+  ![](../images/201/openshift-logs.png)
+
+## watsonx.ai Environment Setup (optional)
+
+If you are planning on governing and evaluating the watsonx.ai generative model as part of **[lab 207](/watsonx/watsonxgov/level-4/207)** and if your Cloud Pak for Data software environment is not equipped with GPUs and doe not have watsonx.ai installed, go through the steps in **[lab 1](/watsonx/watsonxgov/1)** to provision an **IBM watsonx.ai/watsonx.governance SaaS** environment from TechZone. Once the environment is provisioned, continue to **[lab 202: Deploy up a watsonx.ai model](/watsonx/watsonxgov/level-4/202)**
```

Comment:
For this section I would mention that the Geo location **Must be us-south** otherwise it won't be usable to finish lab 207.

Reply:
Can take this as a follow on item.